modules/fundamental/src/advisory/service/test.rs

```diff
@@ -12,27 +12,34 @@ use trustify_cvss::cvss3::{
 };
 use trustify_module_ingestor::graph::advisory::{
     advisory_vulnerability::{VersionInfo, VersionSpec},
-    AdvisoryInformation,
+    AdvisoryContext, AdvisoryInformation,
 };
+
 use trustify_test_context::TrustifyContext;
 
-async fn ingest_and_link_advisory(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
-    let advisory = ctx
-        .graph
+async fn ingest_sample_advisory(
+    ctx: &TrustifyContext,
+    title: String,
```

Comment:
I think if you take a `&str` instead, you can remove all the `.to_string()` calls from the callers. You may have to call `.to_string()` within this function, depending on what `AdvisoryInformation` expects, but better one than many.
```suggestion
    title: &str,
```

Reply:
thanks for the review... but now I'm entering in a weirdo lifetimes thing that I should learn more about : )